src/api/assets/wallets.md

```diff
@@ -41,6 +48,12 @@ single balance and a single currency.
 | accountId | String | The id of the account                                            |
 | ledgerId  | String | The id of the ledger e.g. centrapay.nzd.main, centrapay.nzd.test |
 
+{% h4 Optional Fields %}
+
+| Field      | Type    | Description                                                           |
+|:-----------|:--------|:----------------------------------------------------------------------|
+| settlement | Boolean | Configure the Wallet to be a [Settlement Wallet](#settlement-wallets).|
```

Comment:
Can we add `EXPERIMENTAL` in front of the description?

Reply:
We will need to update the merchant config docs to indicate that only settlement wallets can be provided for nzd payment options.

Reply:
@ncjones have implemented those changes now